<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>790</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Задание 2</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="le_name_language"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_add_language">
          <property name="text">
           <string>Добавить язык программирования</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_delete_language">
          <property name="text">
           <string>Удалить язык программирования</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chckb_icon">
          <property name="text">
           <string>Отображать иконки</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="lv_languages"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbtn_up">
          <property name="text">
           <string>Вверх</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_down">
          <property name="text">
           <string>Вниз</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Задание 3</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>4</y>
       <width>780</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pbtn_change_color">
        <property name="text">
         <string>Сменить цвет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tv_pc"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
